test/battle/hold_effect/eject_pack.c

```diff
@@ -63,3 +63,27 @@ SINGLE_BATTLE_TEST("Eject Pack is triggered by self-inflicting stat decreases")
         ANIMATION(ANIM_TYPE_MOVE, MOVE_CELEBRATE, opponent);
     }
 }
+
+DOUBLE_BATTLE_TEST("Eject Pack will activate before Eject Button since player Wobbuffet is the fastest mon and commander activation will follow")
+{
+    GIVEN {
+        PLAYER(SPECIES_TATSUGIRI) { Speed(10); Ability(ABILITY_COMMANDER); }
+        PLAYER(SPECIES_WOBBUFFET) { Speed(100); Item(ITEM_EJECT_BUTTON); }
+        PLAYER(SPECIES_DONDOZO) { Speed(20); }
+        OPPONENT(SPECIES_WOBBUFFET) { Speed(50); Item(ITEM_EJECT_PACK); }
+        OPPONENT(SPECIES_WOBBUFFET) { Speed(10); }
+        OPPONENT(SPECIES_WYNAUT) { Speed(1); }
+    } WHEN {
+        TURN { MOVE(opponentRight, MOVE_MAKE_IT_RAIN); SEND_OUT(playerRight, 2); SEND_OUT(opponentRight, 2); }
```

Comment:
Why does this test check for `opponentRight`'s activation of Eject Pack at the end if `opponentLeft` is the one with Eject Pack?

Reply:
That's my bad